source/core/parameters_multiphysics.cc

```diff
@@ -114,6 +114,11 @@ Parameters::VOF::declare_parameters(ParameterHandler &prm)
 {
   prm.enter_subsection("VOF");
   {
+    prm.declare_entry("limit force",
```

Comment:
For what I remember, I used a parameter a while ago (`apply_gravity_on_fluid0` or something), but we decided with @blaisb that it should always be `true` so we removed the parameter. Do you have a case where you want it to be false?

Reply:
like, is there a case where you would like to skip mass conservation on a fluid, and still apply the source term on it?

Reply:
oh, or do you want to limit it no matter the density ratio?

Reply:
The idea is that the behavior of not applying the force in one of the fluid is not valid for multiple cases.
I would be for the idea of removing the parameter all together and applying force everywhere.

Reply:
This parameter and limiting force in the VOF solver are removed.